pkgs/applications/misc/openbrf/default.nix

```diff
@@ -1,6 +1,22 @@
 { mkDerivation, lib, stdenv, fetchFromGitHub, qtbase, vcg, glew, qmake, libGLU, libGL }:
 
-
+let
+  # openbrf fails to with newer vcg so use the 2016 one
+  vcg-1_0_1 = vcg.overrideAttrs(final: prev: {
+    version = "1.0.1";
+    src = fetchFromGitHub {
+      owner = "cnr-isti-vclab";
+      repo = "vcglib";
+      rev = "v${final.version}";
+      hash = "sha256-Tvxe88izIxnIZNpTmgfTUExHTsED4JExiiwfmxGTCEo=";
+    };
+    propagatedBuildInputs = [ ];
+    installPhase = ''
+      mkdir -p $out/include
+      cp -r vcg wrap eigenlib $out/include
+    '';
+  });
+in
```

Comment:
I think i prefer the prior fix, with the patches, just add a comment stating upstream is inactive.

Reply:
so after looking at the code more i started to prefer the 1.0.1 version. this package is header only and the code doesn't use a global include path. eg: includes are include "vcg/.../file.h" "eigenlib/Eigen/xxx.h" so is not going to conflict with the other libraries (QT) used by this package.

if i was a user of this app and could test better i would feel better about the update and patch but i am basically just testing the code by compiling, and making sure the app launches which is poor.